EMIP/1-1000/EMIP00634.xml

```diff
@@ -43,7 +43,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
                   <msItem xml:id="ms_i1">
                     <locus from="1r" to="15r" facs="007"/>
-                  <title type="complete" ref="LIT5944CommMystatgogia"/>
+                  <title type="complete" ref="LIT5944Mystagogia"/>

```

Comment:
ok but you could have also simply call your new file LIT5944CommMystatgogia and not change anything here

Reply:
Unfortunately, I have already given a new name. I will do it the easier for the next time.

Reply:
I repeat. You can rename the file. Then this PR can be closed without merging.

Reply:
I renamed the file and closed and deleted this branch.